packages/backend/src/config/app.ts

```diff
@@ -195,6 +195,7 @@ export const Config = {
   walletAddressRedirectHtmlPage: process.env.WALLET_ADDRESS_REDIRECT_HTML_PAGE,
   localCacheDuration: envInt('LOCAL_CACHE_DURATION_MS', 15_000),
   operatorTenantId: envString('OPERATOR_TENANT_ID'),
+  enableIlpTiming: envBool('ENABLE_ILP_TIMING', true),
```

Comment:
```suggestion
  enableIlpTiming: envBool('ENABLE_ILP_TIMING_TELEMETRY', true),
```
I think it's clearer that way